core/core-dotspacemacs.el

```diff
@@ -140,6 +140,10 @@ with `:' and Emacs commands are executed with `<leader> :'.")
 (defvaralias 'dotspacemacs-remap-Y-to-y$ 'evil-want-Y-yank-to-eol
   "If non nil `Y' is remapped to `y$'.")
 
+(defvaralias 'dotspacemacs-ex-substitute-global 'evil-ex-substitute-global
+  "If non nil, :substitute changes all matches in a line by default.
+The 'g' flag toggles one or all matches.")
```

Comment:
If you look at all the docstrings in this file, literals always starts with a back quote and ends with a straight quote, as for instance in:

    If non nil `Y' is remapped to `y$'.

So it's better for consistency to use this convention than the one used in `evil-ex-substitute-global` documentation :-)